ldms/src/ldmsd/ldmsd_prdcr.c

```diff
@@ -561,6 +562,12 @@ static void prdcr_connect_cb(ldms_t x, ldms_xprt_event_t e, void *cb_arg)
 	}
 	switch (e->type) {
 	case LDMS_XPRT_EVENT_CONNECTED:
+		if (prdcr->warned ) {
```

Comment:
just set this to 0. There's no need for a 'connection error cleared' message.

Reply:
We found with v3 users that having a 'cleared' message was useful. They aren't left wondering if something is perpetually broken. If the debug level is DEBUG, then it's obvious, but nobody runs production ldmsd at DEBUG level.

Reply:
@tom95858 @valleydlr There is also the question, good to discuss here, if "connection error" (which is expected in normal cluster operation of aggregators) should be an LERROR or LWARNING level message routinely. 

One could also easily add a producer parameter "warn_after=$seconds" that emits an LERROR level message only after $seconds of connection retry failures. 

This would make the typical case (admin/agg nodes come up before compute nodes) quiet for the frivolous errors caused by the compute nodes still being in their boot phase. Normal operation at 1000 nodes scale shouldn't cause a log storm of frivolous error/warning messages due to variations in daemon boot order.